fullscreen.bs

```diff
@@ -365,7 +362,9 @@ attribute's getter must run these steps:
    <a lt="unfullscreen a document">unfullscreen <var>descendantDoc</var></a>.
    <!-- cross-process -->
 
-   <li><p>For each <var>exitDoc</var> in <var>exitDocs</var>, in order,
+   <li><p>For each <var>exitDoc</var> in <var>exitDocs</var>, in order: If <var>exitDoc</var> is
```

Comment:
Suggestions also welcome here. Would it be better have exitDocs contain only the documents that should fully exit, and then handle the document above separate? (That is currently "If exitDocs’s last document has a browsing context container, append that browsing context container’s node document to exitDocs.")